news/661.api_change

```diff
@@ -0,0 +1 @@
+Unify the CLI parser keyword to dash-style
```

Comment:
This sounds much bigger than what it actually is.
How about:
```suggestion
Hydra argparse flags are now consistently using dash-style.
```

Reply:
done